azure-mgmt-resources/src/main/java/com/microsoft/azure/management/resources/implementation/ResourceManager.java

```diff
@@ -149,7 +149,8 @@ public ResourceManager withSubscription(String subscriptionId) {
     }
 
     private ResourceManager(RestClient restClient, String subscriptionId) {
-        super(restClient, subscriptionId);
+        super(null, subscriptionId);
```

Comment:
oh, ok.
another thing I tried was have an alternative constructor that took an alreadyh created ResourceManager, but that couldn't work inside a ResourceManager constructore...